project/backend/prometheus.yml

```
@@ -0,0 +1,46 @@
+    metrics_path: /actuator/prometheus
+    static_configs:
+      # Target uses the container name defined in api-gateway/docker-compose.yml
+      - targets: ['api-gateway-app:9000'] # App internal port
+    basic_auth:
+      username: 'user'
+      password: 'password'
+  - job_name: 'user-service'
+    metrics_path: /actuator/prometheus
+    static_configs:
+      # Target uses the container name defined in project-manager/docker-compose.yml
+      - targets: [ 'user-service-app:8082' ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)